### ISSUE TITLE
MacOS libfido2 v1.4.0 returns fido_dev_make_cred: FIDO_ERR_RX

### ISSUE THREAD

**Issue by t0b0** [Yubico/libfido2#179]
**What version of libfido2 are you using?**
1.4.0

**What operating system are you running?**
macOS Catalina 10.15.4

**What application are you using in conjunction with libfido2?**
ssh-keygen

**Error occurs with following security keys**
* Yubikey NEO, vendor=0x1050, product=0x0113 (Yubico Yubikey NEO U2F)
* Google Titan USB-A (ca. 2017), vendor=0x096e, product=0x0858 (FT U2F)
* Google Titan USB-A (ca. 2020), vendor=0x096e, product=0x0858 (FT U2F)

**It works with the following keys**
* Yubikey 5, vendor=0x1050, product=0x0406 (Yubico YubiKey FIDO+CCID)
* Yubikey U2F Security Key, vendor=0x1050, product=0x0120 (Yubico Security Key by Yubico)

**FYI, It works with all keys on Ubuntu 20.04 with libfido2 version 1.3.1**

**The error**
After running `ssh-keygen -t ecdsa-sk`, I get the error `Key enrollment failed: invalid format`. 

`ssh-keygen -vvv` gives me: 
```
ssh-keygen -t ecdsa-sk -f titan_ecdsa_sk -vv
Generating public/private ecdsa-sk key pair.
You may need to touch your authenticator to authorize key generation.
debug1: start_helper: starting /usr/local/Cellar/openssh/8.2p1_1/libexec/ssh-sk-helper
debug1: sshsk_enroll: provider "internal", device "(null)", application "ssh:", userid "(null)", flags 0x01, challenge len 0
debug1: sshsk_enroll: using random challenge
debug1: ssh_sk_enroll: using device IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/RP05@1C,4/IOPP/UPSB@0/IOPP/DSB1@1/IOPP/UPS0@0/IOPP/pci-bridge@2/IOPP/pci8086,15f0@0/AppleUSBXHCITR@00000000/AppleUSB20XHCIPort@00100000/USB2.1 Hub@00100000/AppleUSB20Hub@00100000/AppleUSB20HubPort@00130000/USB2.0 Hub@00130000/AppleUSB20Hub@00130000/AppleUSB20HubPort@00133000/U2F@00133000/U2F@0/AppleUserUSBHostHIDDevice
debug1: ssh_sk_enroll: fido_dev_make_cred: FIDO_ERR_RX
debug1: sshsk_enroll: provider "internal" returned failure -1
debug1: ssh-sk-helper: Enrollment failed: invalid format
debug1: ssh-sk-helper: reply len 8
debug1: client_converse: helper returned error -4
Key enrollment failed: invalid format
```

**Comment by martelletto** [comment]
Hi,

This sounds a lot like #166. Would it be possible for you to check whether the fix (39544a2) works for you as well?

-p.

**Comment by t0b0** [comment]
Yes, that fixed it for me, thanks! I submitted the patch to homebrew: https://github.com/Homebrew/homebrew-core/pull/54599
